scania-digital-design-system/sdds
# Fix/textarea textfield maxlength

**Describe pull-request**  
Renamed the textcounter control to max length.

**Solving issue**  
Fixes: AB#2682

**How to test**  
1. Go to storybook link below
2. Check in Components -> Textfield/Textarea
3. Check the controls.

**Screenshots**  
-

**Additional context**  
-
